spec/lang/machine.md

```diff
@@ -62,6 +62,10 @@ struct StackFrame<M: Memory> {
     /// Expresses what happens after the callee (this function) returns.
     return_action: ReturnAction<M>,
 
+    /// The basic block to jump to when the callee calls ResumeUnwind.
+    /// If `None`, UB will be raised when the callee calls ResumeUnwind.
```

Comment:
What is the "callee" here? I don't understand this comment.

Shouldn't this be a new field in `ReturnAction::ReturnToCaller`?